src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/equalshashcode/InputEqualsHashCode.java

```diff
@@ -6,40 +6,40 @@
 
 package com.puppycrawl.tools.checkstyle.checks.coding.equalshashcode;
 
-public class InputEqualsHashCode { // ok
```

Comment:
This input file lgtm....
You will have to remove the suppression for the respective file from [checkstyle-input-suppressions.xml](https://github.com/checkstyle/checkstyle/blob/master/config/checkstyle-input-suppressions.xml)